app/src/components/TokenCard.tsx

```diff
@@ -0,0 +1,26 @@
+
+import React from 'react';
+
+import { Alignment, Box, Direction, Image, PaddingSize, Stack, Text, TextAlignment } from '@kibalabs/ui-react';
+
+import { CollectionToken } from '../client/resources';
+
+export interface TokenCardProps {
+  collectionToken: CollectionToken;
+}
+
+export const TokenCard = (props:TokenCardProps): React.ReactElement => {
+  return (
+    <Stack direction={Direction.Vertical} isFullWidth={true} childAlignment={Alignment.Center} contentAlignment={Alignment.Start} paddingHorizontal={PaddingSize.Wide1}>
```

Comment:
the structure of this component is off. it should be a box with a vertical stack inside it, not the other way around.

Reply:
remove paddding on this it doesnt make sense.